quiz solution..ipynb

```diff
@@ -0,0 +1,137 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "id": "fd48036c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import csv"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "id": "0f568cc7",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "hello i'm neel.\n",
+      "i'm from surat.\n",
+      "i want to be web developer.\n",
+      "i am big fan of hitman.\n",
+      "i'm 18 years old.\n",
+      "mine hobbies are cricket,kabaddi and pool.\n"
+     ]
+    }
+   ],
+   "source": [
+    "with open('C:\\sqlite3\\intro.txt','r') as f:\n",
```

Comment:
how to open this file to another way ?

Reply:
any text editor threw we open this file..